lib/assets/blog-article.mst

```diff
@@ -5,9 +5,9 @@
 </head>
 
 <body>
-<header class="extra-header">
```

Comment:
OK, makes sense. I don't feel strongly one way or the other. Just always watch out with renames and use `git grep extra-header` to make sure nothing else (at least in the same git repository) is depending on the old name.

BTW, for future reference this would be a good candidate for splitting out into its own commit so you can explain in the commit message why the rename was done. No need to go back and change it now though.